File: DKS-API/Models/DKS/DtrFgtShoes.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DKS.API.Models.DKS
{
    public class DtrFgtShoes
    {
        public string? FACTORYID { get; set; }
        [Key]
        public string SAMPLENO { get; set; }
        public string ARTICLE { get; set; }
        public string STAGE { get; set; }
        public string TEST { get; set; }
        public DateTime QC_RECEIVE { get; set; }
        public DateTime? QC_ETD { get; set; }
    }
}

File: DKS-SPA/src/app/core/_models/dtr-fgt-shoes.ts
import { ModelInterface } from "./interface/model-interface";

export class DtrFgtShoes implements ModelInterface{
     factoryid :string;
     sampleno :string;
     article :string;
     stage :string;
     test :string;

     qcReceive :Date;
     qcEtd :Date;
     upday :Date;
}
